Dockerfile

```diff
@@ -1,36 +1,44 @@
-FROM node:8-alpine
+FROM node:alpine AS build
 
 #the only reason why we need git here, is that we have designsystem in another github repo
 RUN apk add --no-cache git
```

Comment:
Does this affect caching of subsequent layers?

Reply:
apk --no-cache means it is not stored in local apk repository
https://stackoverflow.com/questions/49118579/alpine-dockerfile-advantages-of-no-cache-vs-rm-var-cache-apk